modules/cart/src/EventSubscriber/CartLanguage.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\commerce_cart\EventSubscriber;
+
+use Drupal\Core\Language\LanguageManager;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class OrderSubscriber.
```

Comment:
The class description here doesn't really add anything, and it's wrong. 

``Makes a cart language-aware.``